veth-create-host.sh

```diff
@@ -0,0 +1,29 @@
+#! /bin/bash
```

Comment:
How do we remove the links? Will it work if we run:
```
tb_ctl.sh start
tb_ctl.sh stop
tb_ctl.sh start
tb_ctl.sh stop
```

Reply:
yes, it works.

Reply:
So, could you please clarify how the links are get removed?

Reply:
It looks like something removes them. I wasn't able to find any leftovers except symlinks to removed netns... Removing symlinks might be complicated. I'll think about it.

Reply:
I suppose that when parent netns is removed veth is removed automatically. I do not see any leftovers in filed descriptors as well.
What I see - symlinks in /var/run/netns. That's what I'll fix in run.sh.